Calculator component designed

Anand8317 opened the pull request:
- Removed the react boilerplate template to add new features.
- Created a component directory that contains the project's first component Calculator.js
- In Calculator.js rendered a calculator UI whose design matches the design specifications.

KDlamini commented on src/App.js:
```diff
@@ -1,28 +1,8 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
```
Kindly make sure that the `App.js` component is class-based as per project requirements. Although you'll eventually refactor your components to using Hooks, it is important to learn how to use both class and functional components. For now, you'll be learning how to use class-based components until further notice.

KDlamini reviewed (changes requested):
### Hi @Anand8317 🙋‍♀️,  

Great work so far! 💯 
You have done a great job creating the Math magicians components using React. 🎊 
There are some issues that you still need to work on improving your project according to the requirements. You are almost there! 💪🏾 

## Highlights 🌞 

✔️  The project has a directory called `components`
✔️  The `Calculator.js` component is class-based
✔️  The `App.js`(main component) renders the `Calculator.js` component
✔️  No linter errors

## Required Changes ♻️

- [ ] Please ALWAYS remove the Microverse logo and use your project logo if you have any. If the project doesn't have a logo remove the logo entirely.
 
 - [ ] Please try to keep all headers from the [README template](https://github.com/microverseinc/readme-template). The only header you can remove is the FAQ section. If there's no information for a section yet rather leave a "coming soon..." text than delete it. You still need to update each section as you continue building your project and add more features. Your table of contents has a lot of sections removed making it different from the one below.
 
![image](https://user-images.githubusercontent.com/38088818/211853550-e6cb8292-b25d-4ba7-9414-1dafe1b136c0.png)

- [ ] Please make sure that the Pull Request is a short summary describing introduced changes in more detail. You can modify the bullets point in the project requirements and use them to list changes that were introduced in your project.

In your case you could use the following points:

![image](https://user-images.githubusercontent.com/38088818/211857800-f582ab67-df1b-4137-a10e-4c07d5fb2a4e.png)

To create a pull request description as follows:

In this feature, the following feature were implemented.

- Deleted all the boilerplate from CRA (text, images, styles) to make way for new features.
- Created a directory called components and inside with a new Calculator.js file.
- In Calculator.js, rendered a new calculator UI that matches the design specifications.

**Kindly look through the comments below 👇 for required changes**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

- [ ] Since you have a LICENSE file in your project, it would be best if you create a link to your license.

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Anand8317 commented:
Thanks for the detailed review.
Changes I have made under the recommendation provided by CR.
- I have remade the Readme file. 
- In App.js, Changed the functional component to that of class-based.
- Updated the description of the PR.

anagudelogu reviewed (approved):
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi @Anand8317, 
I'm Andres.

You did a great job working on this milestone, your project is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖  
Congratulations! 🎉 

<img src="https://media0.giphy.com/media/BPJmthQ3YRwD6QqcVD/giphy.gif"/>

## Highlights
✅ Great use of Git flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Descriptive PR title, summary and commit messages. 💬 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
